Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackFormConfiguration.swift

```diff
@@ -56,6 +56,7 @@ public class SentryUserFeedbackFormConfiguration: NSObject {
     /**
      * Allows the user to send a screenshot attachment with their feedback.
      * - note: Default: `true`
+     * - warning: Your app must provide a value in its Info plist for the `NSPhotoLibraryUsageDescription` key, or the app will crash upon requesting the necessary `PHPhotoLibrary` authorization.
```

Comment:
`h`: That sounds severe. Could we mitigate this problem with more than a comment? Can't we disable the photo library feature if the user doesn't provide this specific key? We could also log an error message when the user enables this feature and the key isn't in the info.plist file.

Reply:
yes, I think we could check for the existence in the main bundle's user info dictionary before trying to call the APIs.

Reply:
I'm going to do this in a separate PR because I've found some other things that need to be fixed in the photo picker UX.